normandy/base/tests/test_schema.py

```diff
@@ -34,6 +34,18 @@ def test_resolve_user_by_id(self, gql_client):
         res = gql_client.execute(GQ().query.user(id=u.id).fields("username"))
         assert res == {"data": {"user": {"username": u.username}}}
 
+    def test_password_not_queryable(self, gql_client):
+        u = UserFactory()
+        res = gql_client.execute(GQ().query.user(id=u.id).fields("password"))
+        assert res == {
+            "errors": [
+                {
+                    "locations": [{"column": 23, "line": 1}],
```

Comment:
This seems a bit fragile since we are generating the queries. Can we change the `[{"column": 23, "line": 1}]` part to something more flexible? Maybe as simple as `"locations": Whatever()`.